pkg/installer/install.go

```diff
@@ -37,12 +39,21 @@ func (m *manager) Install(ctx context.Context) error {
 		steps.Action(func(ctx context.Context) error {
 			var err error
 			// Applies ARO-specific customisations to the InstallConfig
-			g, err = m.applyInstallConfigCustomisations(ctx, installConfig, image)
+			g, err = m.applyInstallConfigCustomisations(installConfig, image)
 			return err
 		}),
 		steps.Action(func(ctx context.Context) error {
 			return m.persistGraph(ctx, g)
 		}),
+		steps.Action(func(ctx context.Context) error {
+			m.oc.Properties.NetworkProfile.UserDefinedRouting = api.UserDefinedRoutingDisabled
+			if m.oc.Properties.APIServerProfile.Visibility == api.VisibilityPrivate &&
+				m.oc.Properties.IngressProfiles[0].Visibility == api.VisibilityPrivate &&
+				feature.IsRegisteredForFeature(m.sub.Properties, api.FeatureFlagUserDefinedRouting) {
+				m.oc.Properties.NetworkProfile.UserDefinedRouting = api.UserDefinedRoutingEnabled
```

Comment:
can you pull this into it's own function to make things a bit cleaner?

Reply:
+1